src/Components/Books.js

```diff
@@ -12,14 +13,16 @@ const Books = () => {
   const booksData = useSelector((state) => state.books);
   return (
     <div>
-      {Object.keys(booksData).map((bookId) => (
-        <Book
-          key={bookId}
-          title={booksData[bookId][0].title}
-          author={booksData[bookId][0].author}
-          id={bookId}
-        />
-      ))}
+      <div className="overflow-auto" id="scrollBar">
+        {Object.keys(booksData).map((bookId) => (
+          <Book
+            key={bookId}
+            title={booksData[bookId][0].title}
+            author={booksData[bookId][0].author}
+            id={bookId}
+          />
+        ))}
+      </div>
```

Comment:
- [ ] Update font-style according to design
![image](https://user-images.githubusercontent.com/62705947/186641807-df335eed-d795-47b7-8d9a-daf84a88dbb4.png)

Reply:
- [ ] Font size should be properly given just as on the design
![image](https://user-images.githubusercontent.com/62705947/186642371-f76e704f-f290-422a-ae7e-b3f812a3388d.png)